packages/fxa-auth-server/pm2.config.js

```diff
@@ -6,12 +6,13 @@ module.exports = {
   apps: [
     {
       name: 'auth',
-      script: './bin/key_server.js',
+      script: 'ts-node -P tsconfig.json bin/key_server.js',
```

Comment:
Does it not work without the `-P` here?  I thought `tsconfig.json` is the default value.

Reply:
good thinking